README.md

```diff
@@ -1,33 +1,76 @@
-# Tool Migration
+# Canvas LTI Tool Migration
 
-## Getting started
+This application migrates LTI tools in a Canvas instance from one version to another.
 
-### Installation
+## Running with Docker compose
+
+This is the recommended way to run the application.  Docker compose will ensure the application is run in a consistent environment with the correct version of Python and all the required dependencies.
+
+### Prerequisites
+* [Docker Desktop](https://www.docker.com/products/docker-desktop/) — This must be installed first.
+* [tool-migration](https://github.com/tl-its-umich-edu/tool-migration/releases/latest) — Download the latest release of this app and unzip it.  The latest release will be at the top of the list.  The zip file is found under the heading "Assets".  (Alternatively, the repository can be cloned.)
+
+### Configuration
+
+In a shell (e.g., using Terminal.app on macOS), navigate to the directory where the app was unzipped or cloned.  Then copy the sample configuration file:
+
+```shell
+cp .env.sample .env
+```
+
+Then open the newly copied `.env` file with an editor and fill in the missing values.  E.g., to edit with TextEdit.app on macOS:
+
+```shell
+open -a TextEdit .env
+```
+
+In the future, there may be other ways to configure, such as a CLI, prompts, or files.
```

Comment:
Not sure whether we want to do those tasks in the future. We will see the need from BSAs in the migration cycle.

Reply:
Resolved in 2073a47.